src/UI/Button.elm

```diff
@@ -492,6 +515,7 @@ hyperlinkView cfg size width body action =
                 :: Font.regular
                 :: Font.underline
                 :: Element.pointer
+                :: (id |> Maybe.map ElementUtils.id |> Maybe.withDefault Element.pointer)
```

Comment:
@PedroHLC is there a better way of doing this?

Reply:
https://github.com/PaackEng/paack-ui/blob/15f172d45eec94c0eeff0568c4b183b293403859/src/UI/Internal/Basics.elm#L5